### DISCUSSION TITLE
版本选择让用户自主创建收藏夹创建收藏夹

### DISCUSSION THREAD

**Discussion by Raclom** [Meloong-Git/PCL#400] [提问] [answered]
希望可以在版本选择可以自建收藏夹，虽然可以该版本的名字，还有收藏了版本的收藏夹以外，想某些下载版本较多的可以自定义添加收藏夹和自定义名称文件夹

**Comment by WForst-Breeze** [comment, marked as answer]
省流：暂时不会制作。

您好！很高兴为您解答\(￣︶￣*\))
针对此问题，我曾采访过龙猫本人**类似**的问题，如下：
### Q：![1](https://user-images.githubusercontent.com/110760354/202454118-19ae7994-7c1e-4040-8c8f-78d4fe13a9e4.png)

### A：![2](https://user-images.githubusercontent.com/110760354/202454181-42a8cb3a-953c-4c21-98d4-7a0d88d65fba.png)
